packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.module.css

```diff
@@ -0,0 +1,37 @@
+.createTeamDialog {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space--24);
+}
+
+.createTeamForm {
```

Comment:
Y U NO .root? Also should this be the first thing in the file?

Reply:
Hihi 👍